src/app/App/component.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+
+import { AppWrapper } from './styles';
+
+type AppProps = {};
+
+const App: React.FC<AppProps> = () => {
+  return (
+    <Container>
+      <AppWrapper>Hello World!</AppWrapper>
+    </Container>
```

Comment:
let's place `Container` inside `AppWrapper` component, as `Footer` component should be placed outside container. (because of it's full-screen background color), like

```tsx
<AppWrapper>
    <Container>Hello World!</Container>
    <Footer />
</AppWrapper>
```

Reply:
It was added only to show that we could use Bootstrap. It is not a real layout. If you tell that we want to add some default `font-family` then `AppWrapper` should be parent component, because there are initial styles.

Reply:
got it 👍  

so, let's make `AppWrapper` as a parent component. and create some base structure here, as many PR's are now currently pulled in from this one, and we should have **one source of truth** for it.

I suggest:
```tsx
<AppWrapper>
      {/*<Header />*/}
      <Container>App Content</Container>
      {/*<Footer />*/}
</AppWrapper>
```

Reply:
As I saw it will be changed here https://github.com/MecenatOrg/frontend/pull/8/files

Reply:
yes, but your PR will be merged first. and all our's are pulled from yours.
does it make sense?

Reply:
Removed container, it is not needed here